### PR TITLE
`print`->`fail` and suppress the warning in another case.

### PR DESCRIPTION
**Pull request by thomasvl**

### CODE REVIEW

File: lib.bzl [hunk 1]
```diff
@@ -27,6 +27,8 @@ load("//lib:types.bzl", _types = "types")
 load("//lib:unittest.bzl", _asserts = "asserts", _unittest = "unittest")
 load("//lib:versions.bzl", _versions = "versions")
 
+# There isn't a better way to issue warnings in Starlark, so use print.
+# buildifier: disable=print
```
Comment (laurentlb) [inline lib.bzl]: use fail and remove the rest of the file

File: lib/versions.bzl [hunk 2]
```diff
@@ -93,6 +93,8 @@ def _check_bazel_version(minimum_bazel_version, maximum_bazel_version = None, ba
                 minimum_bazel_version,
             ))
         elif not native.bazel_version:
+            # There isn't a better way to issue warnings in Starlark, so use print.
+            # buildifier: disable=print
             print("Current Bazel is not a release version; cannot check for compatibility. " +
                   "Make sure that you are running at least Bazel {}.".format(minimum_bazel_version))
```
Comment (laurentlb) [inline lib/versions.bzl]: remove line instead

File: skylark_library.bzl [hunk 3]
```diff
@@ -20,6 +20,8 @@ Deprecated:
 
 load("//:bzl_library.bzl", "StarlarkLibraryInfo", "bzl_library")
 
+# There isn't a better way to issue warnings in Starlark, so use print.
+# buildifier: disable=print
 print(
     "WARNING: skylark_library.bzl is deprecated and will go away in the future, please" +
     " use bzl_library.bzl instead.",
```
Comment (laurentlb) [inline skylark_library.bzl]: use fail

Comment (thomasvl) [reply skylark_library.bzl]: Done

File: lib/versions.bzl (hunk 2 above)
Comment (thomasvl) [reply lib/versions.bzl]: Done

File: lib.bzl (hunk 1 above)
Comment (thomasvl) [reply lib.bzl]: Done

File: skylark_library.bzl (hunk 3 above)
Comment (laurentlb) [reply skylark_library.bzl]: nit. `skylark_library.bzl has been removed, please`

File: lib.bzl [hunk 4]
```diff
@@ -14,36 +14,8 @@
...
-load("//lib:shell.bzl", _shell = "shell")
-load("//lib:structs.bzl", _structs = "structs")
-load("//lib:types.bzl", _types = "types")
-load("//lib:unittest.bzl", _asserts = "asserts", _unittest = "unittest")
-load("//lib:versions.bzl", _versions = "versions")
-
-print(
-    "WARNING: lib.bzl is deprecated and will go away in the future, please" +
+fail(
+    "lib.bzl is deprecated and will go away in the future, please" +
```
Comment (laurentlb) [inline lib.bzl]: nit. `lib.bzl has been removed, please`

Comment (thomasvl) [reply lib.bzl]: Done

**Pull request merged**